HW_1/task8.py

```diff
@@ -0,0 +1,16 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек 
+# отломить k долек,если разрешается сделать один разлом по прямой между дольками 
+# (то есть разломить шоколадку на два прямоугольника).
+
+# *Пример:*
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n = int(input("Input m\n"))
+m = int(input("Input n\n"))
+k = int(input("Input k\n"))
+if k <= n * m and (k % n == 0 or k % m == 0):
```

Comment:
Можно немного расширить первое условие по нижней границе допустимых значений переменной `k`.
```suggestion
if 0 < k < n * m and (k % n == 0 or k % m == 0):
```